ui/ui-expenses/src/screens/prices/capture/capture.container.js

```diff
@@ -0,0 +1,15 @@
+import { container } from 'ui-hocs'
+import Component from './capture'
+
+
+const mapStore = (store, ownProps, { camera, window }) => ({
+  device: camera.getCameras()[0],
+  capture: camera.getCapture(),
+  useCamera: camera.useCamera,
+  takeCapture: camera.takeCapture,
+  cancel: camera.removeCapture,
+  height: window.innerHeight / 3,
```

Comment:
If you don't make styles, remove this.
This is bad practice

Reply:
this props it's not only to style.
The component `Capture` use a canva, and this size is to define the size of capture. Not just the size of component.